rocketpy/rocket/aero_surface/generic_surface.py

```diff
@@ -17,12 +17,7 @@ def __init__(
         self,
         reference_area,
         reference_length,
-        cL=0,
-        cQ=0,
-        cD=0,
-        cm=0,
-        cn=0,
-        cl=0,
+        coefficients="all_null",
```

Comment:
With this change, there is no need to make the `coefficients` parameter optional. Lets just leave it mandatory and remove the `"all_null"` option.

Reply:
If it becomes mandatory, then we would have to pass coefficients even if we wanted the default with everything zero. Is it okay?

Reply:
At least one coefficient should not be zero for the class to be useful, otherwise it doesn't do much. So I think this is the best way